gtda/externals/python/ripser_interface.py

```diff
@@ -253,22 +275,20 @@ def ripser(X, maxdim=1, thresh=np.inf, coeff=2, metric="euclidean",
             # nonzero births
             dm = sparse.coo_matrix(dm)
 
-    if sparse.issparse(dm) or collapse_edges:
-        if collapse_edges:
-            if not sparse.issparse(dm):
-                row, col, data = \
-                    gtda_collapser.flag_complex_collapse_edges_dense(dm,
-                                                                     thresh)
-            else:
-                coo = dm.tocoo()
+    is_sparse = sparse.issparse(dm)
+    if is_sparse or collapse_edges:
+        if is_sparse:
+            coo = dm.tocoo()
```

Comment:
I don't think it's necessary to assign the value to a variable, but we should directly forward it to the function if it isn't use anywhere else.

Reply:
You're right!